charts/teamcity/templates/agent-daemonset.yaml

```diff
@@ -38,6 +38,39 @@ spec:
       serviceAccountName: {{ include "teamcity.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.agent.podSecurityContext | nindent 8 }}
+      {{- if .Values.volumePermissions.enabled }}
+      initContainers:
+      - name: init-chown-data
+        image: "{{ .Values.volumePermissions.image.image }}:{{ .Values.volumePermissions.image.imageTag }}"
+        imagePullPolicy: "{{ .Values.volumePermissions.image.pullPolicy }}"
+        resources:
+          limits:
+            cpu: 50m
+            memory: 32Mi
+          requests:
+            cpu: 50m
+            memory: 32Mi
+        command:
+          - sh
+          - -c
+          - |
+            chown -Rv {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} /data
```

Comment:
shouldn't we follow symlinks with `-h`?

Reply:
as far as I saw there are no symlinks in these dirs